src/main/java/core/basesyntax/service/StorageServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import core.basesyntax.controller.ControllerDao;
+import core.basesyntax.controller.ControllerDaoImpl;
+import core.basesyntax.model.CreateStrategy;
+import core.basesyntax.model.CreateStrategyImpl;
+import core.basesyntax.model.Fruit;
+import java.time.LocalDate;
+import java.util.List;
+
+public class StorageServiceImpl implements StorageService {
+    private ControllerDao<Fruit> controllerDao = new ControllerDaoImpl();
+    private CreateStrategy<Fruit> createStrategy = new CreateStrategyImpl();
+
```

Comment:
```suggestion
    private ControllerDao<Fruit> controllerDao = new ControllerDaoImpl();
    private CreateStrategy<Fruit> createStrategy = new CreateStrategyImpl();
    
    public StorageServiceImpl(ControllerDao controllerDao, CreateStrategy createStrategy) {
               this.controllerDao = controllerDao;
               this.createStrategy = createStrategy;
    }
```
    з точку зору написання коду, тут можна ще таке застосувати DI паттерн
Аналогічний приклад з того, що Богдан сьогодні розбирав на лекції
https://github.com/mate-academy/jv-jul20-practice/blob/master/src/mate/academy/solid/BikeParserStrategy.java